### PR TITLE
Install page updated

### LINKED ISSUE #79: Improve clarity of which environment files are to be used during installation

### PR DESCRIPTION
**Pull request by DevarshiChoudhury**
Fixes #79

### CODE REVIEW

**Review by drannawatts — APPROVED** [review]
Looks good to me!

File: docs/source/install.rst [hunk 1]
```diff
@@ -323,6 +336,19 @@ C flags for compilation of the X-PSI extensions. Because the library location
...
+
+.. code-block:: bash
+
+    cd examples/examples_fast/Modules/
+    python main.py
+
+This module performs a ``likelihood check``. If the likelihood value calculated matches
+the given value, X-PSI is functioning as expected, else it will raise an error message.
+The module will then initiate sampling using MultiNest (assuming that it's installed),
+and given the settings, it should take ~15 minutes. To cancel mid-way press ``ctrl + C``.
```
Comment (thjsal) [inline docs/source/install.rst:350]: In the end, it seems that the test-run takes now only 5 minutes! Can you modify accordingly?

**Review by thjsal — COMMENTED** [review]
The test-run checking if installation works takes actually now 5 min instead of the mentioned 15 min. Perhapds should be corrected in the instructions?

File: docs/source/install.rst (hunk 1 above)
Comment (DevarshiChoudhury) [reply docs/source/install.rst]: Done

**Pull request merged**